repos/rust-lang/rust-project-goals.toml

```diff
@@ -14,4 +14,5 @@ leadership-council = "maintain"
 libs = "maintain"
 libs-api = "maintain"
 project-edition-2024 = "maintain"
```

Comment:
this is the conflict that needs to be rebased away, it's been since changed to `edition = "maintain"`